families/supplychain/python/sawtooth_supplychain/rest_api/route_handlers.py

```diff
@@ -148,7 +164,7 @@ def __init__(self, loop, db_cnx=None):
 
         try:
             rows = await db_query(self._db_cnx, query, query_tuple)
-        except psycopg2.OperationalError as e:
+        except psycopg2.OperationalError:
             LOGGER.debug("Could not connect to database.")
```

Comment:
Changed this and all other 500 errors to LOGGER.exception rather than LOGGER.debug